ppr-ui/src/views/newMhrRegistration/MhrReviewConfirm.vue

```diff
@@ -101,7 +101,7 @@
               <h3>Other Information</h3>
             </v-col>
             <v-col cols="9">
-              <p>{{ getMhrRegistrationOtherInfo || '(Not Entered)' }}</p>
+              <p v-html="getMhrRegistrationOtherInfo.replace(/\r?\n/g, '<br />') || '(Not Entered)'"></p>
```

Comment:
Try `.trim` on the component i.e. ` v-model.trim="otherRemarks"`

If this is the result of a different set of ux feedback, please create a new computed property so we can keep the regex out of the template. 

Also, what is the purpose here? This is purely for review because this formatting wouldn't have any impact on whats sent in the filing, or be reflective in the outputs, if that was the designs intention.

Reply:
Reworked to add HTML formatted string to the store model (for filing outputs and Review page template).

Reply:
Removed adding Html to the store. We might need a further discussion on this. For now, I will merge.